manifests/e/EchoLink/EchoLink/2.3.121.1/EchoLink.EchoLink.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: EchoLink.EchoLink
+PackageVersion: 2.3.121.1
+PackageLocale: en-US
+Publisher: EchoLink.org

```

```suggestion
Publisher: Synergenics, LLC
```
This should match control panel